commons/connector/fitbit/fitbit_food_log.avsc

```diff
@@ -0,0 +1,108 @@
+{
+  "namespace": "org.radarcns.connector.fitbit",
+  "type": "record",
+  "name": "FitbitFoodLog",
+  "doc": "A summary of the user's food log entry for a given day.",
+  "fields": [
+    {
+      "name": "time",
+      "type": "double",
+      "doc": "Device timestamp in UTC (s)."
+    },
+    {
+      "name": "timeReceived",
+      "type": "double",
+      "doc": "Time that the data was received from the Fitbit API (seconds since the Unix Epoch)."
+    },
+    {
+      "name": "isFavorite",
+      "type": "boolean",
+      "doc": "Whether the food is the favorite one."
+    },
+    {
+      "name": "logId",
+      "type": "long",
+      "doc": "The id of the logged food."
+    },
+    {
+      "name": "accessLevel",
+      "type": "string",
+      "doc": "The access level of the food."
+    },
+    {
+      "name": "amount",
+      "type": "int",
+      "doc": "The amount of the food."
+    },
+    {
+      "name": "brand",
+      "type": "string",
+      "doc": "The brand of the food."
+    },
+    {
+      "name": "foodId",
+      "type": "long",
+      "doc": "The id of the food."
+    },
+    {
+      "name": "locale",
+      "type": "string",
+      "doc": "The locale of the food."
+    },
+    {
+      "name": "mealTypeId",
+      "type": "long",
+      "doc": "The meal type id of the food."
+    },
+    {
+      "name": "name",
+      "type": "string",
+      "doc": "The name of the food."
+    },
+    {
+      "name": "unitId",
```

Comment:
Can you please document where this unit ID comes from? For example, is it from a list or a library or repository? What is the URL for that?

Reply:
Yes from here https://dev.fitbit.com/build/reference/web-api/nutrition/get-food-units/

Reply:
should be OK... could you add that to the doc string?

Reply:
```
Additional Information
Typically, an application retrieves the complete list of units once at startup. From then on, the application will receive a food-specific list of unit ids along with the [Favorite](https://dev.fitbit.com/build/reference/web-api/nutrition/get-favorite-foods/), [Recent](https://dev.fitbit.com/build/reference/web-api/nutrition/get-recent-foods/), or [Frequent](https://dev.fitbit.com/build/reference/web-api/nutrition/get-frequent-foods/) food or by making a query to [Search Foods](https://dev.fitbit.com/build/reference/web-api/nutrition/search-foods/) endpoint.

These are the IDs of units that apply to the favorite food. The application will refer to the complete list of units retrieved previously to get and display to the user the unit name or plural name for each unit ID on food-specific list.
```